media/css/_navigation.scss

```diff
@@ -0,0 +1,127 @@
+.navbar-inner {
+  background: $sidebar-background;
+  border-width: 0 !important;
+}
+
+.navbar-inverse {
+  .navbar-inner {
+    background: $sidebar-background;
+  }
+
+  .brand:hover,
+  .nav > .active > a,
+  .nav > .active > a:hover,
+  .nav > .active > a:focus {
+    background: $sidebar-link-highlight;
+  }
+}
+
+.navbar {
+  .brand {
+    margin-left: 3px;
+    margin-right: 3px;
+    max-height: 50px;
+    overflow: visible;
+    padding: 0 3px;
+  }
+
+  .nav {
+    display: inline;
+    font-size: 14px;
+    list-style: none;
+    margin: 0;
+    text-align: left;
+
+    li {
+      &:hover {
+        background: $sidebar-link-highlight;
+        color: $white;
+      }
+    }
+
+    > li {
+      > ul {
+        -webkit-box-shadow: 1px 6px 8px rgba(0,0,0,.175);
+        box-shadow: 1px 6px 8px rgba(0,0,0,.175);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.175)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.175` should be written with a leading zero as `0.175`<br>Commas in function arguments should be followed by a single space